argusclient/client.py

```diff
@@ -416,7 +416,18 @@ def add(self, alert):
             alertobj.trigger = alertobj.triggers.add(alert.trigger)
         if alert.notification:
             alertobj.notification = alertobj.notifications.add(alert.notification)
-        if alert.trigger and alert.notification:
+        elif alert.notifications and isinstance(alert.notifications, list):
+            notifications = []
+            for notification in alert.notifications:
+                notifications.append(alertobj.notifications.add(notification))
+            alertobj.notifications = notifications
+        if alert.trigger and alert.notifications:
```

Comment:
Isn't the logic here same as the existing logic for a single notification? You should be able to condense both into one loop (i.e., if you initialize a list with a single notification).